Task_Position_the_Cubes/index.css

```diff
@@ -1,27 +1,65 @@
-
-body {
-  background-color: lightblue;
-}
-p{
```

Comment:
The 'center' value for the 'margin' property is invalid. You should use 'auto' or specify the top and bottom margins separately.